File: MuayeneYonetimPortali/MuayeneYonetimPortali.Web/Modules/ServerTypes/Administration/TwoFactorCodeColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { TwoFactorCodeRow } from "./TwoFactorCodeRow";

export interface TwoFactorCodeColumns {
    Id: Column<TwoFactorCodeRow>;
    Username: Column<TwoFactorCodeRow>;
    Code: Column<TwoFactorCodeRow>;
    ExpireTime: Column<TwoFactorCodeRow>;
}

export class TwoFactorCodeColumns extends ColumnsBase<TwoFactorCodeRow> {
    static readonly columnsKey = 'Administration.TwoFactorCode';
    static readonly Fields = fieldsProxy<TwoFactorCodeColumns>();
}

File: MuayeneYonetimPortali/MuayeneYonetimPortali.Web/Modules/Administration/TwoFactorCode/TwoFactorCodeColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace MuayeneYonetimPortali.Administration.Columns;

[ColumnsScript("Administration.TwoFactorCode")]
[BasedOnRow(typeof(TwoFactorCodeRow), CheckNames = true)]
public class TwoFactorCodeColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public string Username { get; set; }
    [EditLink]
    public string Code { get; set; }
    public DateTime ExpireTime { get; set; }
}